src/components/Search/SearchBar.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useState } from 'react';
+import TextInput from 'ink-text-input';
+import { Text, Box, useFocus, useInput, useFocusManager } from 'ink';
+import { BrowserSessionContext } from '../../BrowserSessionProvider';
+import { SearchResults } from './SearchResults';
+
+export const SearchBar = () => {
+  const { isFocused } = useFocus({ id: 'search-bar'});
+  const { focus, focusPrevious } = useFocusManager();
+  const session = useContext(BrowserSessionContext);
+  const [inputValue, setInputValue] = useState('');
+  const [ searchResultActive, setSearchResultActive ] = useState(false);
+
+  useInput((_, key) => {
+    if (!isFocused || searchResultActive) return;
+
+    if (key.upArrow) {
+      setSearchResultActive(false);
+      focusPrevious();
+    }
+  });
```

Comment:
should we add controls for focusNext? What if we later add something after search (like an Exit button?)? Maybe arrow down if searchResultActive is false will focusNext?